src/pages/FlssPrototypes/SampleLoanDisplay.vue

```diff
@@ -0,0 +1,1159 @@
+<template>
+	<www-page-minimal
+		:header-theme="headerTheme"
+		:footer-theme="footerTheme"
+		:flss-logo-url="flssLogoUrl"
+	>
+		<div class="flss-landing">
+			<kv-loading-overlay
+				v-if="loadingPage"
+				class="flss-landing__loading-page"
+			/>
+
+			<!-- TODO: Alter CTA if Checkout is ready -->
+			<flss-hero
```

Comment:
Please just comment out this component for now (lines 14-18). We would want to reuse one of the existing hero components rather than create a new one.